docs/reference/modules/remote-cluster-network.asciidoc

```diff
@@ -0,0 +1,102 @@
+[[remote-cluster-network-settings]]
+==== Advanced remote cluster (API key based model) settings
+
+beta::[]
+
+Use the following advanced settings to configure the remote cluster interface (API key based model)
+independently of the <<transport-settings,transport interface>>. You can also
+configure both interfaces together using the <<common-network-settings,network settings>>.
+
+`remote_cluster_server.enabled`::
+(<<static-cluster-setting,Static>>, boolean)
+Determines whether the remote cluster server should be enabled. This setting must
```

Comment:
Nit: would add a little more detail because we never really define what a remote cluster server is. Something along the lines of "Enables the remote cluster server interface. This allows the cluster to serve API-key based cross cluster requests."

Reply:
This should have a link to the new remote cluster page once #98330 is merged. I also added a brief description as you suggested.